README.md

```diff
@@ -1,59 +1,121 @@
-# Fluence JS
+# Fluence JS Client
 
-[![npm](https://img.shields.io/npm/v/@fluencelabs/fluence)](https://www.npmjs.com/package/@fluencelabs/fluence)
+[![npm](https://img.shields.io/npm/v/@fluencelabs/js-client.api?label=@fluencelabs/js-client.api)](https://www.npmjs.com/package/@fluencelabs/js-client.api)
+[![npm](https://img.shields.io/npm/v/@fluencelabs/js-client.web.standalone?label=@fluencelabs/js-client.web.standalone)](https://www.npmjs.com/package/@fluencelabs/js-client.web.standalone)
 
-Official TypeScript implementation of the Fluence Peer.
+This is the Javascript client for the [Fluence](https://fluence.network) network.
 
-## Getting started
 
-To start developing applications with Fluence JS refer to the official [documentation](https://fluence.dev/docs/build/fluence-js/)
+## Get Started
 
-## Contributing
+Adding the Fluence JS client for your web application is very easy:
+
+1. Add a script tag with the JS Client bundle to your `index.html`. The easiest way to do this is using a CDN (like [JSDELIVR](https://www.jsdelivr.com/) or [UNPKG](https://unpkg.com/)). The script is large, thus we highly recommend to use the `async` attribute.
+
+    Here is an example using the JSDELIVR CDN:
+
+    ```html
+    <head>
+      <title>Cool App</title>
+      <script src='https://cdn.jsdelivr.net/npm/@fluencelabs/js-client.web.standalone@0.10.0/dist/js-client.min.js'
+        async></script>
+    </head>
+    ```
+
+    If you cannot or don't want to use a CDN, feel free to get the script directly from the `npm` package and host in yourself: `https://www.npmjs.com/package/@fluencelabs/js-client.web.standalone`. You can find the script in the `/dist` directory of the package. (Note: this option means that developers understand what they are doing and know how to serve this file from their own web server.)
```

Comment:
This note was for us. IMO we should either re-phrase it to me more soft or get rid of it completelty.

Also suggest to  hide the full link:

```
[@fluencelabs/js-client.web.standalone](https://www.npmjs.com/package/@fluencelabs/js-client.web.standalone)
```